### PR TITLE
Add instructions for running smaller sets of e-commerce tests

### PR DESCRIPTION
**Pull request by JoyceZhu**
Documents commands to run only Python tests, only Python unit tests in a specific file, or only JavaScript tests in e-commerce. The current e-commerce docs currently only describe how to run the full suite of unit tests or quality checks. 
### Reviewers
- [x] Subject matter expert: @douglashall 
- [x] Doc team review (sanity check/copy edit/dev edit):  @pdesjardins 
### Testing
- [x] Ran ./run_tests.sh without warnings or errors

### CODE REVIEW

File: en_us/install_operations/source/ecommerce/test_ecommerce.rst [hunk 1]
```diff
@@ -60,6 +60,32 @@ When you create E-Commerce tests, use the ``TestCase`` class in
...
+  substitute the desired file path.
+
+  .. code-block:: bash
+
+      $ DISABLE_ACCEPTANCE_TESTS=True ./manage.py test
+      ecommerce.courses.tests.test_utils --settings=ecommerce.settings.test
+      --with-ignore-docstrings --logging-level=DEBUG
+
+  Adding the ``DISABLE_MIGRATIONS=1`` option will decrease the time needed to
+  run tests at the cost of skipping migrations.
```
Comment (douglashall) [inline en_us/install_operations/source/ecommerce/test_ecommerce.rst]: I would rephrase this to something like "Setting the DISABLE_MIGRATIONS=1 variable will significantly decrease the time needed to run the tests by creating the test database directly from Django model definitions as opposed to using the defined migrations." And then follow it with a code block example.

**Comment by douglashall** [comment]
One comment. Otherwise looks good! 👍

File: en_us/install_operations/source/ecommerce/test_ecommerce.rst [hunk 2]
```diff
@@ -60,6 +60,32 @@ When you create E-Commerce tests, use the ``TestCase`` class in
 ``Partner`` objects configured. This will help you test any code that relies on
 these models, which are used for multi-tenancy.
 
+* To run all Python unit tests and quality checks, run the following command.
```
Comment (pdesjardins) [inline en_us/install_operations/source/ecommerce/test_ecommerce.rst:63]: On the doc team, we wrap lines with newlines at 80 characters. We use a Sublime text editor function to do that (https://openedx.atlassian.net/wiki/display/DOC/Documentation+Tools#DocumentationTools-AddingPackagestoSublimeText). So we don't see long lines like this one. Minor point, but that's our doc rule.

File: en_us/install_operations/source/ecommerce/test_ecommerce.rst [hunk 3]
```diff
@@ -60,6 +60,32 @@ When you create E-Commerce tests, use the ``TestCase`` class in
...
+
+  .. code-block:: bash
+
+      $ make validate_python
+
+* To run all Python unit tests and quality checks **in parallel**, run the
```
Comment (pdesjardins) [inline en_us/install_operations/source/ecommerce/test_ecommerce.rst]: We typically don't emphasize using bold or italics. I'm not sure I agree with that rule. Maybe just change these to single asterisks for italics?

**Comment by pdesjardins** [comment]
I added two formatting comments. 👍

File: en_us/install_operations/source/ecommerce/test_ecommerce.rst (hunk 2 above)
Comment (JoyceZhu) [inline en_us/install_operations/source/ecommerce/test_ecommerce.rst:63]: Thanks for the info! FYI I just tried wrapping this line with the plugin, which says it's 78 characters -- does it look too long on GitHub?

**Comment by pdesjardins** [comment]
@JoyceZhu maybe it's just the display on my screen. I'm seeing wrapping. Don't worry about it though. 👍 👍 👍
